akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -151,11 +118,9 @@ object Sink extends SinkApply {
    * element is dropped and the stream continues.
    *
    * @see [[#mapAsyncUnordered]]
-   */
+   */ // FIXME Why is this method even useful?
```

Comment:
because users asked for it. There will be always a pressure between keeping things minimal, and having convenience methods. The usual trend is that users like the convenience methods, so we add a couple of them if there are enough people asking.